docs/data/README.md

```diff
@@ -0,0 +1,11 @@
+# Data Module Reference
```

Comment:
This file was previously: `docs/designers-developers/developers/data/README.md`. It looks like GitHub didn't pick up this rename.